1.8/usage/gpu-support.md

```diff
@@ -0,0 +1,59 @@
+---
+post_title: GPU Acceleration
+nav_title: GPU Acceleration
+feature_maturity: experimental
+menu_order: 110 
+---
+
+You can configure your DC/OS service to use the GPUs (Graphics Processing Units) of your nodes. GPUs can dramatically accelerate big data workloads. [Learn more](http://www.nvidia.com/object/what-is-gpu-computing.html).
+
+# Configure your Cluster for GPUs
+
+Follow these steps to enable GPU support on your cluster. 
+
+1. Add the `enable_gpu_isolation: true` flag to your `config.yaml` when you install DC/OS on your cluster. If you are installing DC/OS from the AWS cloud template, follow [the instructions here](/docs/1.8/administration/installing/cloud/aws/advanced/aws-custom/). Otherwise, follow [the custom installation instructions](/docs/1.8/administration/installing/custom/).
```

Comment:
I would reverse these two steps. They should first ensure that they have the libraries installed, and then add the configuration flag to `config.yaml` and install DC/OS on their cluster.